pom.xml

```diff
@@ -106,6 +106,11 @@
                     <version>8.0.30</version>
                     <scope>runtime</scope>
                 </dependency>
+                <dependency>
+                    <groupId>com.aliyun</groupId>
+                    <artifactId>aliyun-encdb-mysql-jdbc</artifactId>
```

Comment:
Since this dependency isn't required by standard MySQL instances, I think it should go in its own profile.  Something like this (pseudocode, please fill in and test the details):

```xml
<profile>
  <id>aliyundb</id> <!-- FIXME: Is this the best name to use here? -->
  ...
  <!-- copies of most of the rest of the MySQL profile dependencies -->
  <!-- additional aliyun db dependencies -->
                  <dependency>
                    <groupId>com.aliyun</groupId>
                    <artifactId>aliyun-encdb-mysql-jdbc</artifactId>
                  </dependency>
...
</profile>
```